### PR TITLE
Util function to avoid mistakes+overflows when fetching chain data

### PR DESCRIPTION
**Pull request by haydenshively**
Sometimes the health computed by the borrower lens was larger than Javascript could handle, resulting in Prime page crashing.

Also, the health on the earn Borrow page was not being divided by 1e18.

I added a new util function to make things clearer. Both issues are fixed now.

### CODE REVIEW

File: prime/src/data/MarginAccount.ts [hunk 1]
```diff
@@ -226,10 +202,10 @@ export async function fetchMarginAccount(
       assets: assets,
       liabilities: liabilities,
       sqrtPriceX96: toBig(oracleResult[0]),
-      health: health.div(1e9).toNumber() / 1e9,
+      health: health,
       lender0: lender0,
       lender1: lender1,
-      iv: oracleResult[1].div(1e9).toNumber() / 1e9,
+      iv: toImpreciseNumber(oracleResult[1], 18),
```
Comment (IanWoodard) [inline prime/src/data/MarginAccount.ts:208]: Could we move this up with the health declaration to stay consistent?

**Review by IanWoodard — APPROVED** [review]

**Comment by IanWoodard** [comment]
> Sometimes the health computed by the borrower lens was larger than Javascript could handle, resulting in Prime page crashing.
> 
> Also, the health on the earn Borrow page was not being divided by 1e18.
> 
> I added a new util function to make things clearer. Both issues are fixed now.

I thought health was at most 10,000.

**Comment by haydenshively** [comment]
Health is hardcoded to be 10000 when liabilities are 0. But when liabilities are very very small (non-zero), it can be larger.

**Review by IanWoodard — APPROVED** [review]
LGTM

**Pull request merged**